buildpack/telemetry/fluentbit.py

```diff
@@ -122,6 +124,17 @@ def _await_logging_endpoint():
         splunk.print_failed_message()
 
 
+def _set_up_environment(model_version, runtime_version):
```

Comment:
I liked this approach. Pretty awesome 💯 
We just need to pass the required metadata attributes as environment vars to the fluentbit process. Neat!